backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -88,7 +88,7 @@ const revisionAdded = async ({ teamspace, project, model, revId, modelType, cali
 			{ _id: 0, tag: 1, author: 1, timestamp: 1, desc: 1, rFile: 1, format: 1, statusCode: 1, revCode: 1 });
```

Comment:
![image](https://github.com/user-attachments/assets/fbb083c3-1519-4e22-a400-4f05c6757f76)
Can we set the SVG drawing to have a white background so it looks like a 2D sheet and also it won't be hindered by users setting a custom background colour

Reply:
To clarify, the end result should have the area highlighted in red being white (approx) and the area outside of the red boundary should be the current background colour